vagrant/requirements.txt

```diff
@@ -22,4 +22,4 @@ psycopg2>=2.6.1,<2.7
 wrapt>=1.10.8,<1.11
 # only needed for ldap caching
 ldap3==2.1.0
-celery==4.0.2
+celery==3.1.25
```

Comment:
You should use [`~=`](https://www.python.org/dev/peps/pep-0440/#compatible-release).